### PR TITLE
C code style format and checks

### PR DESCRIPTION
**Pull request by PFigs**
This commit updates the travis pipeline to
check violations of the c code standard.

It also adds a script to apply the style to
any .c or .h file in the project.

### CODE REVIEW

File: python_transport/wirepas_gateway/dbus/c-extension/dbus_c.c [hunk 1]
```diff
@@ -29,16 +27,8 @@ static int on_packet_received(sd_bus_message *m,
...
-                            "tuuyyuyy",
-                            &timestamp_ms,
-                            &src_addr,
-                            &dst_addr,
-                            &src_ep,
-                            &dst_ep,
-                            &travel_time,
-                            &qos,
-                            &hop_count);
+    r = sd_bus_message_read(
```
Comment (GwendalRaoul) [inline python_transport/wirepas_gateway/dbus/c-extension/dbus_c.c]: In this case, could you remove clang formatting?
All those parameters are more readable with one on each line

File: python_transport/wirepas_gateway/dbus/c-extension/dbus_c.c [hunk 2]
```diff
@@ -102,14 +92,13 @@ static int on_packet_received(sd_bus_message *m,
...
-static PyObject* infiniteEventsLoop(PyObject* self, PyObject* args)
+static PyObject * infiniteEventsLoop(PyObject * self, PyObject * args)
 {
     int r;
     /* Release the GIL. It will be acquire by the callback
      * It make this thread totally independant from python code
      */
-    Py_BEGIN_ALLOW_THREADS
-    for (;;)
+    Py_BEGIN_ALLOW_THREADS for (;;)
```
Comment (GwendalRaoul) [inline python_transport/wirepas_gateway/dbus/c-extension/dbus_c.c]: Why is it put on same line?

File: sink_service/source/data.c [hunk 3]
```diff
@@ -140,16 +134,8 @@ static bool onDataReceived(const uint8_t * bytes,
...
-                              "tuuyyuyy",
-                              timestamp_ms,
-                              src_addr,
-                              dst_addr,
-                              src_ep,
-                              dst_ep,
-                              travel_time,
-                              qos,
-                              hop_count);
+    r = sd_bus_message_append(
```
Comment (GwendalRaoul) [inline sink_service/source/data.c]: Same here, it would be better to keep one on each line

File: sink_service/source/main.c [hunk 4]
```diff
@@ -84,7 +82,8 @@ int main(int argc, char *argv[])
                 break;
             case 'i':
                 /* Get the sink id to generate service name */
-                sink_id = strtoul(optarg, NULL, 0);;
+                sink_id = strtoul(optarg, NULL, 0);
+                ;
```
Comment (GwendalRaoul) [inline sink_service/source/main.c]: Can be removed ;-)

File: sink_service/source/otap.c [hunk 5]
```diff
@@ -151,31 +146,29 @@ static int process_scratchpad(sd_bus_message * m,
...
     /* Read only parameters backup-ed with a table (Read each time stack starts) */
-    SD_BUS_PROPERTY("StoredLen",      "u", NULL, offsetof(sink_otap_t, stored_len), 0),
-    SD_BUS_PROPERTY("StoredCrc",      "q", NULL, offsetof(sink_otap_t, stored_crc), 0),
-    SD_BUS_PROPERTY("StoredSeq",      "y", NULL, offsetof(sink_otap_t, stored_seq), 0),
-    SD_BUS_PROPERTY("StoredStatus",   "y", NULL, offsetof(sink_otap_t, stored_status), 0),
-    SD_BUS_PROPERTY("StoredType",     "y", NULL, offsetof(sink_otap_t, stored_type), 0),
-    SD_BUS_PROPERTY("ProcessedLen",   "u", NULL, offsetof(sink_otap_t, processed_len), 0),
-    SD_BUS_PROPERTY("ProcessedCrc",   "q", NULL, offsetof(sink_otap_t, processed_crc), 0),
-    SD_BUS_PROPERTY("ProcessedSeq",   "y", NULL, offsetof(sink_otap_t, processed_seq), 0),
+    SD_BUS_PROPERTY("StoredLen", "u", NULL, offsetof(sink_otap_t, stored_len), 0),
```
Comment (GwendalRaoul) [inline sink_service/source/otap.c]: I prefer the old allignement here

File: python_transport/wirepas_gateway/dbus/c-extension/dbus_c.c (hunk 1 above)
Comment (PFigs) [reply python_transport/wirepas_gateway/dbus/c-extension/dbus_c.c]: It actually makes no sense shy the format is being applied like this :(

**Pull request merged**